.devcontainer/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.177.0/containers/typescript-node/.devcontainer/base.Dockerfile
+
+# [Choice] Node.js version: 16, 14, 12
+ARG VARIANT="16-buster"
+FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:0-${VARIANT}
+
+# [Optional] Uncomment this section to install additional OS packages.
+# RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+#     && apt-get -y install --no-install-recommends <your-package-list-here>
+
+# Install an additional version of node using nvm
+ARG EXTRA_NODE_VERSION=14
+RUN su node -c "source /usr/local/share/nvm/nvm.sh && nvm install ${EXTRA_NODE_VERSION}"
+
+# Install more global node packages
+RUN su node -c "npm install -g npm@7.13 lerna pm2"
```

Comment:
lerna and pm2 are local deps why are they installed globally?

Reply:
The difference between local and global npm-packages is not really relevant with this container-technique, because the container is built only for the vivid-planet-repo.

The reason why I install them "globally" is because I can install them during the docker-image build-process. The image is built only once (typically); so those packages are installed only once at image-build-time, and not every time the container is started. (So it's just a time saver!)

Reply:
But of course, lerna and pm2 is installed from the package.json anyway, so those global packages are not needed.